infrastructure/lambda/api.yaml

```diff
@@ -393,7 +393,14 @@ components:
         currentResidency:
           type: "boolean"
           example: true
-
+#  securitySchemes:
+#    api_key:
+#      type: "apiKey"
+#      name: "x-api-key"
+#      in: "header"
+#
+#security:
+#  - api_key: []
 
```

Comment:
☝️ this is commented out until clients start sending their api keys in the http header. Note that this security covers all resources on this API